site/guides/consumer/function/README.md

```diff
@@ -164,14 +164,14 @@ Here, rather than specifying `gcr.io/kpt-functions/label-namespace` using the
   [functions concepts] page.
 - Learn more ways of using the `kpt fn` command from the [reference] doc.
 
-[container runtime]: ../../producer/functions/container/
-[executables]: ../../producer/functions/exec/
-[starlark scripts]: ../../producer/functions/starlark/
+[container runtime]:/guides/producer/functions/container/
+[executables]:/guides/producer/functions/exec/
+[starlark scripts]:/guides/producer/functions/starlark/
 [label-namespace]: https://github.com/GoogleContainerTools/kpt-functions-sdk/blob/master/ts/hello-world/src/label_namespace.ts
 [VCS]: https://en.wikipedia.org/wiki/Version_control
-[function producer docs]: ../../producer/functions/
+[function producer docs]:/guides/producer/functions/
```

Comment:
how comes some links are absolute and some are relative? Do you want to standardize on one?

Reply:
Definitely want to standardize absolute links. I'll take a pass at catching some more in addition to the recent commits.

Reply:
Fixed what seemed to be ~25 files of this.